### PR TITLE
Updates to fr and tn m3u files

### PR DESCRIPTION
**Pull request by dct-infra**
- Added descriptions
- Using recommended url format for streamlink px
- Added 1 channel

### CODE REVIEW

File: channels/al.m3u [hunk 1]
```diff
@@ -11,6 +11,8 @@ http://us.bestvideostreaming.is/8737/playlist.m3u8
 https://shkoder.gjirafa.com/api/media/rgjirafa/t0110y/index.m3u8
 #EXTINF:-1 tvg-id="Cufo.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/cjVAXnZ.png" group-title="Kids",Çufo (576p)
 http://93.157.62.180/Cufo/index.m3u8
+#EXTINF:-1 tvg-id="EuronewsAlbania.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/hTBah1z.jpg" group-title="News",Euronews Albania (1080p)
```
Comment (LaneSh4d0w) [inline channels/al.m3u:14]: Ehh... isn't Euronews french ?

https://fr.wikipedia.org/wiki/Euronews
https://en.wikipedia.org/wiki/Euronews

**Comment by Sphinxroot** [comment]
yup sorry

**Comment by Sphinxroot** [comment]
here if we dont have them to list

#EXTINF:-1 ,Euronews Albania (AL)
https://www.youtube.com/channel/UChR-A__NS_C5kHDWj3PeAhw/live
#EXTINF:-1 ,Euronews Deutsch (DE)
https://www.youtube.com/channel/UCACdxU3VrJIJc7ujxtHWs1w/live
#EXTINF:-1 ,Euronews Italiano (IT)
https://www.youtube.com/channel/UC1mX9vuLOYf8fhaXS_KcDRg/live
#EXTINF:-1 ,euronews español (ES)
https://www.youtube.com/channel/UCyoGb3SMlTlB8CLGVH4c8Rw/live
#EXTINF:-1 ,Euronews português (PT)
https://www.youtube.com/channel/UCUmEPYxmnyQDeRUcFkslmQw/live
#EXTINF:-1 ,Euronews русски (RU)
https://www.youtube.com/channel/UCFzJjgVicCtFxJ5B0P_ei8A/live
#EXTINF:-1 ,Euronews ελληνικά (GR)
https://www.youtube.com/channel/UC8HJZNfDPoySgW9DN3YGNaA/live
#EXTINF:-1 ,Euronews magyarul (HU)
https://www.youtube.com/channel/UC4Ct8gIf9f0n4mdyGsFiZRA/live
#EXTINF:-1 ,Euronews Francais (FR) (GEO)
https://www.youtube.com/channel/UCW2QcKZiU8aUGg4yxCIditg/live
#EXTINF:-1 ,Euronews English (UK) (GEO)
https://www.youtube.com/channel/UCSrZ3UV4jOidv8ppoVuvW9Q/live
#EXTINF:-1 ,Euronews  ჯორჯია (GE)
https://live2.tvg.ge/eng/EURONEWSGEORGIA/playlist.m3u8

**Comment by LaneSh4d0w** [comment]
The issue with the Albanian stream is that they're streaming 2 things at once. YouTube always takes the most recent livestream.

**Comment by Sphinxroot** [comment]
> The issue with the Albanian stream is that they're streaming 2 things at once. YouTube always takes the most recent livestream.

do streamlink handle multiple live stream from the same channel id ?

in kodi we can add &live=1 ect  for the plugins

**Comment by LaneSh4d0w** [comment]
> 
> 
> > The issue with the Albanian stream is that they're streaming 2 things at once. YouTube always takes the most recent livestream.
> 
> do streamlink handle multiple live stream from the same channel id ?
> 
> in kodi we can add &live=1 ect for the plugins

Not quite sure about it, since the redirection is made by YouTube, not Streamlink.

**Comment by dct-infra** [comment]
resolving conflicts

**Pull request closed without merging**